### ISSUE TITLE
Dynamic langnames.json endpoint

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/translationDatabaseWeb#173]
For tS and uW, we could use a dynamic API endpoint that provides a list of only the newly created languages since the date stamp provided.

For example, tS has a langnames.json file from 2015-07-07 and it asks tD for any new languages since then, hence, it would receive a list of maybe 4 languages, instead of the whole shebang.

This could either take the form of a URL parameter or a JSON request.

**Comment by vleong2332** [comment]
@jag3773 Just checking to see if the need and the priority is still valid.

**Comment by jag3773** [comment]
@vleong2332 Yes, it would make requests much faster and easier for clients in low-bandwidth situations (like tS and tR).  I think this came up in our last discussion about adding new languages.
